### PR TITLE
Make TwitterMap Customizable

### PR DESCRIPTION
**Pull request by HotLemonJuice**
Related issue: #353 

Main change:
- Now we can change `application.conf` file to remove search bar and customize predefined keywords
- Predefined keywords will appear ***on the top of the page*** via the drop-down menu

# Demos
1 mov
2 mov
3 mov
4 mov

### CODE REVIEW

**Comment by HotLemonJuice** [comment]
Hi @chenlica I have addressed your comments.

**Comment by HotLemonJuice** [comment]
@chenlica @JavierJia Now the keywords are in the drop-down menu. Check it out.

**Comment by chenlica** [comment]
@HotLemonJuice It LGTM!

**Comment by HotLemonJuice** [comment]
@JavierJia I'm a bit busy for the following few days. Will a few gif pictures do the trick?

**Comment by JavierJia** [comment]
that will also work :-)

**Comment by HotLemonJuice** [comment]
@JavierJia I have uploaded the gifs. Check it out :)

**Comment by JavierJia** [comment]
LGMT 👍

**Comment by chenlica** [comment]
@HotLemonJuice good job!

File: twittermap/conf/application.conf [hunk 1]
```diff
@@ -78,8 +78,16 @@ bounded-mailbox {
 
 cloudberry.register = "http://localhost:9000/admin/register"
 cloudberry.ws = "ws://localhost:9000/ws"
+
+# TwitterMap Extra Function Knobs
+# Sentiment Analysis
 sentimentEnabled = false
 sentimentUDF = "twitter.`snlp#getSentimentScore`(text)"
+# Customize TwitterMap for a specific domain
+removeSearchBar = true
```
Comment (JavierJia) [inline twittermap/conf/application.conf]: @HotLemonJuice can you make it `false` by default?  and then show some gif to see if the UI is the same as before?

**Pull request merged**